projects/gedankenblitze/index.md

```diff
@@ -0,0 +1,72 @@
+--- 
+authors: 
+  - "Julian Broocks"
+authors-url: 
+  - "https://github.com/hulianbrox"
+layout: project
+title: "Gedankenblitze"
+type: project
+---
+![Gedankenblitze](splash.png)
+
+## The Assignment
+
+The assignment was to create an algorithm that had to be sketched on paper by 10 test persons. Content of the algorithm was a simple drawing instruction after defining parameters like tools, format, time and material. The description should be precise and easy to execute while leaving room for free interpretation. Further on, more iterations were conducted by changing parameters of the algorithm to improve its description and output.
+
+"Analog algorithm" was the first assignment of "Gestalten In Code – the basic of generative design" by Fabian Morón Zirfas at the University of Applied Sciences Potsdam.
+
+## My Inspiration 
+Inspiration came from the shapes of lightning, which can be recreated with paper by crumpling it, and from an artwork by Anders Hoff, who published a study of generative algorithms. The centralized arrangement of the elements in the picture were really appealing. So this aspect was implemented into the algorithm. 
+
+![Differnitalmesh3D](./assets/images/inconvergent.png)
+copyrighted by [http://inconvergent.net](http://inconvergent.net/) 
+
+## The Algorithm 
+
+> ### Material and Tools 
```

Comment:
Yeah we will see. Jekyll does not use the original markdown renderer from Gruber